### DISCUSSION TITLE
Save labeled videos (Bonsai with DeepLabCut)

### DISCUSSION THREAD

**Discussion by PikaPei** [bonsai-rx/bonsai#1338] [Q&A]
I know I can use `CsvWriter` & `VideoWriter` to save the video and coordinates of labeled points.
But can I save the labeled videos?

Actually, I saw the discussion here: https://groups.google.com/g/bonsai-users/c/vvPGVt6gVxo
Just wondering if there's another way to do that?

**Comment by bruno-f-cruz** [comment]
Something like this perhaps?

https://groups.google.com/g/bonsai-users/c/e1VAC0AP_lc/m/fiOP1bugAAAJ
